PCbuild/readme.txt

```diff
@@ -52,6 +52,42 @@ Release
     settings, though without PGO.
 
 
+Building Python using Clang/LLVM
+--------------------------------
+
+See https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170
+how to install and use clang-cl bundled with Microsoft Visual Studio.
+You can use the IDE to switch to clang-cl for local development,
+but because this alters the *.vcxproj files, the recommended way is
+to use build.bat:
```

Comment:
```suggestion
See https://learn.microsoft.com/en-us/cpp/build/clang-support-msbuild?view=msvc-170
for how to install and use clang-cl bundled with Microsoft Visual Studio.
You can use the IDE to switch to clang-cl for local development,
but because this alters the *.vcxproj files, the recommended way is
to use build.bat:
```